task/sast-snyk-check/0.1/sast-snyk-check.yaml

```diff
@@ -48,21 +48,9 @@ spec:
         grep -q "$SKIP_MSG" stdout.txt || test_not_skipped=$?
         if [[ "$SNYK_EXIT_CODE" -eq 0 ]] || [[ "$SNYK_EXIT_CODE" -eq 1 ]]; then
           cat sast_snyk_check_out.json
-          HACBS_TEST_OUTPUT=$(jq -rce --arg date $(date +%s) \
-            '{ result: (if (.runs[].results | length > 0) then "FAILURE" else "SUCCESS" end),
-                   timestamp: $date,
-                   namespace: "default",
-                   successes: 0,
-                   note: "",
-                   failures: (.runs[].results | length)
-                 }' sast_snyk_check_out.json || true)
+          HACBS_TEST_OUTPUT=
```

Comment:
I think this file is missing the `. /utils.sh` call.

Reply:
Yes, thanks!

Reply:
Updated